Integrations/integration-PagerDuty.yml

```diff
@@ -157,14 +163,13 @@ script:
             'ReadableContentsFormat': formats['markdown'],
             'HumanReadable': tableToMarkdown(table_name, outputs, USERS_ON_CALL_NOW_HEADERS),
             'EntryContext': {
-                'PagerDutyUser': context
+                'PagerDutyUser(true)': contexts
```

Comment:
@ronykoz why is the condition equals to true? should be `PagerDutyUser(val.ID==obj.ID)`

Reply:
Same as above